lib/rules/jsx-no-target-blank.js

```diff
@@ -64,13 +66,21 @@ module.exports = {
     const configuration = context.options[0] || {};
     const enforceDynamicLinks = configuration.enforceDynamicLinks || 'always';
 
+    const elements = new Map((DEFAULTS.concat(context.settings.linkComponents || [])).map(value => {
```

Comment:
i feel like we could make a separate helper to get the "array of components"?

Reply:
Done.